public/components/getting_started/components/getting_started_collectData.tsx

```diff
@@ -138,7 +144,7 @@ export const CollectAndShipData: React.FC<CollectAndShipDataProps> = ({
   // Auto-select first collector if nothing is selected and a collection method is set
   useEffect(() => {
     if (collectorOptions.length > 0 && !specificMethod && collectionMethod) {
-      handleSpecificMethodChange([{ value: collectorOptions[0].value }]);
+      handleSpecificMethodChange([{ ...OTEL_LOGS_OPTION }]);
```

Comment:
The json used for the Logs / Metrics / Traces are separate. This will cause selecting Metrics or Traces to display the wrong workflow. 

![Screenshot 2024-10-14 at 10 27 16 AM](https://github.com/user-attachments/assets/73a4adbb-bda6-4994-90f2-09f62310b1a7)